### ISSUE TITLE
EnableF8BootMenu conflict

### ISSUE THREAD

**Issue by pagex** [Disassembler0/Win10-Initial-Setup-Script#128]
I've enabled this option on Windows 10 1807, then laptop did not boot with mouse inserted. BIOS post passes, when windows is about to be loaded it just hangs. Had to boot without mouse inserted, Disable this F8 option and boot works fine now.

**Comment by pagex** [comment]
1. Windows PRO was installed manually (laptop came with Win Home)
2. no multi-boot
3. Logitech G300s in HP Spectre x360 (note any other mouse plugged in - boot works, if 2 things are plugged in like USB stick and mouse - boot works)
4. will do

**Comment by Disassembler0** [comment]
I've managed to find a friend with G300s. We tried it with 4 different devices, but weren't able to replicate the problem anywhere (none of them was a HP device, though). Seems like you've won the lottery and the issue is specific to your configuration.

I went once again through possible causes, but I really don't see anything. Basically the only difference between  *legacy* and *standard* boot policy is that *legacy* permits you to interrupt the boot sequence with F8 keypress at any time whereas *standard* displays the menu only when there was a startup failure or when you're booting from recovery or installation media. The rest is completely the same. Same OS loader, same kernel with the same parameters, same HAL, same hardware profile, same drivers, same everything.

You can try to enable logging as described here - https://superuser.com/questions/1140309/windows-10-log-info-when-starting - and see if you can identify the driver or action on which it hangs. There is also a possibility that there's something wrong with your RAM which manifests like this, but I find it unlikely as you would probably notice more weird problems. Other that that, I can suggest only voodoo or exorcist, depending on your preferences.

I'll close the issue as I'm now pretty certain that there is nothing I can fix on my side, but feel free to update, if you find anything worth sharing.

**Comment by r3incarnat0r** [comment]
TL;DR: try to update BIOS first.
---
I would suggest to do BIOS/UEFI update first. I've already solved or reduced a number weird boot-related issues just by doing that. Also try to fiddle with boot or USB options in BIOS/UEFI, even with those which does not seem to be related on first sight. Unfortunately manufacturer's implementation of various BIOS/UEFI stuff is not always perfect, sometimes even far from perfect and updating won't help. Regarding HP, I can name two examples I recently encountered: ProBook 6570b - unable to detect or boot from SSD until SATA mode switched from 6Gbps to 3Gbps, EliteBook 8470p - UEFI and two SSDs - first with Linux, second with Windows, impossible to just simply list them in boot menu and select boot source, had to switch to legacy BIOS completely. Disclaimer - this comment suggests making BIOS/UEFI changes which may lead to serious problems, try it only if you know exactly what you are doing.
